client/components/ReviewRating/ReviewList/Helpfulness.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import axios from 'axios';
+
+const Helpfulness = ({ review, fetchReviews }) => {
+  const onChangeHandler = (e) => {
+    e.preventDefault();
+    const action = e.target.name;
+    axios.put(`/reviews/${review.review_id}/${action}`);
```

Comment:
would we want to fetch reviews inside the 'then' block of this axios put request so as to include the post that was just added?

Reply:
sure, that might be the reason for the delay. thanks.